### ISSUE TITLE
[A11y] Explore Collections - Grid - Mobile - Text Spacing

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1898]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

The text clipping and overlap problems get noticeably worse in a mobile view when changing text spacing to give extra space between letters.

### Suggested Fix

This might not be a WCAG failure, and is likely to be addressed when other tickets are fixed, so this should be low priority.  Just something to pay attention to.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Get a browser about the size of a phone by resizing and/or zooming - verify it looks like your phone's screen.  Then use the Accessibility Insights text spacing test (Adaptable Content -> Text Spacing) to see if text is fully visible after adding spacing.

**Comment by CGillen** [comment]
Improved by other tickets
